common/lib/xmodule/xmodule/word_cloud_module.py

```diff
@@ -247,6 +253,7 @@ def get_html(self):
             'element_id': self.location.html_id(),
             'num_inputs': self.num_inputs,
             'submitted': self.submitted,
+            'js_includes': js_includes,
```

Comment:
@andy-armstrong: Could you advise here? The primary goal of this PR is to remove the heavy JS from WordCloud out of the XModule JS bundle. A secondary goal would be to shift WordCloud to use XBlock views handling more completely so it's not in some mutant half-way state on the front end.